app/src/main/java/com/example/tasky/agenda/domain/model/AgendaPhoto.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.tasky.agenda.domain.model
+
+sealed class AgendaPhoto(val url: String) {
+    class Local(url: String) : AgendaPhoto(url)
```

Comment:
I personally make all classes that inherit from sealed class, a data class
Not just a regular class

Reply:
Yes, but I want to use the variable created in the father class AgendaPhoto and dont want to each of the classes that inherit AgendaPhoto has a new variable that wont be used since I will only use the variable url from AgendaPhoto

Reply:
You could do it like this:
```
sealed class AgendaPhoto(open val url: String) {
    data class Local(override val url: String): AgendaPhoto(url)
}
```
If you don't need the extras a data class gives you, there's no reason trying to force having a data class IMO